Sources/Datadog/CrashReporting/CrashContext/CrashContextProvider.swift

```diff
@@ -10,40 +10,78 @@ import Foundation
 internal protocol CrashContextProviderType: class {
     /// Returns current `CrashContext` value.
     var currentCrashContext: CrashContext { get }
-    /// Notifies on current `CrashContext` change.
+    /// Notifies on `CrashContext` change.
     var onCrashContextChange: ((CrashContext) -> Void)? { set get }
 
     /// Updates the `CrashContext` with last `RUMEvent<RUMViewEvent>` information.
     func update(lastRUMViewEvent: RUMEvent<RUMViewEvent>)
```

Comment:
Now all but this values are delivered to `CrashContextProvider` using publish / subscribe way. TBH, I'd vote for doing the additional change (small refactoring) and making the `lastRUMViewEvent` being an observed value as well. This will clear the `CrashContextProviderType` so it has only two methods:
```swift
internal protocol CrashContextProviderType: class {
   var currentCrashContext: CrashContext { get }
   var onCrashContextChange: ((CrashContext) -> Void)? { set get }
}
```

WDYT @buranmert @acgh ?

Reply:
Done in https://github.com/DataDog/dd-sdk-ios/pull/422/commits/a244076025768f7569180b6bf73be37a72b919dd ✅